File: api/examples/vb.net/BUBBLE_LAUNCH.vb
<Serializable()>
Public Class BUBBLE_LAUNCH
    Public ID As Long = 0
    Public IsInstance As Integer = 0
    Public Flags As Integer = 0
    Public Password As String = String.Empty
    Public DNSID As Long = 0
    Public ProxyList As New List(Of Long)
    Public VPNID As Long = 0
End Class

File: api/examples/c#/BUBBLE_LAUNCH.cs
using System;
using System.Collections.Generic;

[Serializable()]
public partial class BUBBLE_LAUNCH
{
    public long ID = 0L;
    public int IsInstance = 0;
    public int Flags = 0;
    public string Password = string.Empty;
    public long DNSID = 0L;
    public List<long> ProxyList = new List<long>();
    public long VPNID = 0L;
}